client/my-sites/checkout/checkout/style.scss

```diff
@@ -1148,29 +1148,6 @@
 	font-style: italic;
 }
 
-.is-section-checkout.is-jetpack-site {
```

Comment:
Removing all this doesn't break the appearance of old checkout, does it? Important to double check since this PR keeps old checkout running for jetpack in production. I'm leaving this comment here to make sure we test it.

Reply:
Ok, I tested and I verified that the production checkout does match the screenshots in the PR description, so I guess this is known and intentional. 👍